framework/packages/abstract-core/src/native/module_factory.rs

```diff
@@ -63,22 +46,25 @@ pub enum ExecuteMsg {
         version_control_address: Option<String>,
     },
     /// Install modules
-    InstallModules { modules: Vec<ModuleInstallConfig> },
+    InstallModules {
```

Comment:
This call will become unavailable for managers from a previous version. These managers will need to update to be able to install new apps.